### DISCUSSION TITLE
デザイン実装について教えてほしいです

### DISCUSSION THREAD

**Discussion by sayyyyyy** [jphacks/C_2213#22] [Q&A]
- 現在のfigmaのデザインで制作してもいいのか
- レスポンシブ対応は勝手に決めていいのか
- どのファイルのデザインやればいいか

**Comment by guchio33** [comment]
1　デザインのままお願いします！
2    レスポンシブは、スマホ画面は今後変える可能性があります！
3　Top.vue,signIn.vue,singup.vue,Notification.vueはデザイン先にお願いします

**Reply by sayyyyyy** [reply]
了解です！

**Comment by Rei-Suzuki1729** [comment]
結局sassでバリバリ書く感じですか？？
一応簡単な通話機能っぽいもの作ったときにvuetifyをつかったので、vuetifyの使い心地はまあまあな感じで良くも悪くもベータ版という感じでした。

**Reply by sayyyyyy** [reply]
使った方が結果的に早く終わりそうですよね

**Reply by Rei-Suzuki1729** [reply]
https://zenn.dev/uedayou/articles/9e8bd227a13e1c

これをみながら実装していました。

**Reply by sayyyyyy** [reply]
ありがとうございます
使う方向でいきましょう！

**Reply by Rei-Suzuki1729** [reply]
了解です
バージョンは3.0.0-betaでいいですか？？

**Reply by sayyyyyy** [reply]
はい大丈夫です！
